src/content/quickstart/_index.en.md

```diff
@@ -1,17 +1,68 @@
 +++
-title = "Quickstart"
+title = "Quickstart guide"
 date = 2020-02-19T20:03:44+01:00
 weight = 5
 pre = "<b>1. </b>"
 +++
 
+## Basic overview
 
-# Quickstart guide
+Submariner has two main core pieces (the broker and submariner), more information about 
+this topic can be found in the [Architecture](../architecture) section.
 
-## Installing
+#### The broker
+It's an API, to which all participating clusters are given access, and where two objects are exchanged via CRDs:
+* Cluster(.submariner.io): defines a participating cluster and it's IP CIDRs.
+* Endpoint(.submariner.io): defines a connection endpoint to a Cluster, and the reachable cluster IPs from such endpoint.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+This part should be deployed on one K8s cluster, a participating cluster could be the broker as
```

Comment:
```suggestion
The broker should be deployed on one K8s cluster which can be any participating cluster as
long as all other clusters can access its API.
```

Reply:
Your phrasing sounds more confusing to me. I’m trying to make clear that it could be/or could be not participating on the data plane connectivity.

Reply:
How about _“The broker should be deployed on one K8s cluster which can be any cluster as long as …”_

Reply:
Also your phrasing seems to imply that, if it's a participating cluster, it must be accessible to all others, but it's not clear if that also applies to a non-participating cluster. Logically it should. Maybe I'm splitting hairs here but I think _"which can be any cluster as long as …"_ keeps it simple and conveys the proper meaning.

Reply:
May be I'm also splitting hairs, but still your original phasing also makes me think that it has to be one of the dataplane connected clusters.

Reply:
When I find something confusing, I tend to split sentences up ;-).

The broker must be deployed on a single Kubernetes cluster. This cluster’s API server must be reachable by all Kubernetes clusters connected by Submariner. It can be a dedicated cluster, or one of the connected clusters.

Reply:
Thanks, I like the suggestion @skitt . Let me incorporate it, @tpantelis how does it look to you?